### PR TITLE
Overview Enhancement : Issue #21

### PR DESCRIPTION
**Pull request by guilyx**
Issue #21 PR

### CODE REVIEW

**Comment by guilyx** [comment]
Do you feel like this is okay ? Struggled for the example but feels like it's clear enough ?

**Comment by guilyx** [comment]
@sampreets3 :smile:

File: _posts/2020-04-27-project.md [hunk 1]
```diff
@@ -17,17 +17,48 @@ The aim of the project is to determine how real-time capabilities can be brought
...
+- Missing a deadline is considered a system **failure**.
+- It is mandatory for **safety** or **mission-critical** systems.
+
+<p align="center">
+  <img width="500" height="400" src="img/rt_constraints.png">
+</p>
+
+On the top figure, each bar represent a task and the time at which it's executed. The red line represents a deadline. If a task is executed after the deadline, it is considered - in hard real-time systems - a failure. It has a different behaviour for firm or soft real-time systems. For firm constraints, a task that was executed after the deadline will not cause a system failure, but its behavior (i.e. computations) won't be considered. For soft constraints, the behavior will be considered with a non-critical warning.
+
+## Why do we need it ?
```
Comment (sampreets3) [inline _posts/2020-04-27-project.md:34]: The example is spot on. Especially the focus on the design aspect of things definitely helps a lot

**Review by sampreets3 — APPROVED** [review]
It looks good to me :smile:

**Comment by sampreets3** [comment]
merging the PR right away

**Comment by sampreets3** [comment]
hey @guilyx, can you just check if the image you uploaded is accessible or not, because on the webpage, it is not loading. Not an urgent issue by any means, but do take a look at it when you have the time.

**Pull request merged**